book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publish_date, publisher, cover_state, archived: false)
+    super(publish_date, archived: archived)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?
+    super || cover_state == 'bad'
+  end
```

Comment:
Inside this method, you should use the instance variable "@cover_state" instead of "cover_state"